src/foam/nanos/notification/services

```diff
@@ -113,3 +113,51 @@ p({
   "serviceClass":"foam.nanos.notification.email.POP3EmailService",
   "client":"{ \"class\":\"foam.nanos.notification.email.ClientPOP3EmailService\", \"delegate\": { \"class\":\"foam.box.HTTPBox\", \"url\":\"service/pop3\" } }"
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"localSmsMessageDAO",
+  "serviceScript":"""
+    return new foam.dao.EasyDAO.Builder(x)
```

Comment:
Please set this up like the emailMessageDAO - with fixed sized support so we don't load all SMSs into memory.